public/resources/scss/stats.scss

```diff
@@ -899,46 +905,32 @@ a.additional-player-stat:hover {
 }
 
 .stat-intelligence,
-.color-intelligence {
+.stat-magic-find,
+.stat-fishing-speed,
+.color-intelligence
+.color-magic-find,
+.color-fishing-speed {
   color: var(--§b);
 }
 
 .stat-sea-creature-chance,
-.color-sea-creature-chance {
+.color-sea-creature-chance,
+[class $= "wisdom"] {
```

Comment:
What did this compile to?

Reply:
Everything with a class that ends in wisdom (not sure about the spaces though).

@metalcupcake5 I think it's much much safer to specify each type of wisdom, I would do that

Reply:
Oh I thought this was a fancy sass thing I didn't realize that was built in CSS.

One issue is that this actually only selects things where the last class ends in wisdom

I think would be better to select based on actual classes instead of based on the class attribute

Reply:
got it